hastexo/tutor-contrib-s3
# feat: allows configuring AWS_QUERYSTRING_AUTH

## Description

The changes included in this pull request allow configuring  `AWS_QUERYSTRING_AUTH: false` while keeping the default behavior of `AWS_QUERYSTRING_AUTH: true`

## Testing instructions

1. Setup a new instance using this branch of the plugin
2. Add `S3_QUERYSTRING_AUTH: false` to the instance `config.xml`
3. Deploy the instance
4. Check URLs are generated without authentication signature
5. Set the AWS bucket as public
6. Ensure the S3 object URLs generated by the platform are working
